core/src/wallet/tezos/explorers/NodeTezosLikeBlockchainExplorer.cpp

```diff
@@ -69,8 +69,9 @@ namespace ledger {
 
         Future<std::shared_ptr<BigInt>>
         NodeTezosLikeBlockchainExplorer::getFees() {
+            bool parseNumbersAsString = true;
```

Comment:
+1 for putting that in a variable. `json(true)` would sound very weird to me, but `json(parseNumbersAsString)` is okay in that case.